docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
>  I believe that we need to have the following scopes of configuration variables now: application ==> job ==> task

+1 to Yi's suggestion here. Let's distinguish between "application" and "job" level properties. 

> We were going to introduce a new "processors" scope that uses the same syntax as this "streams" scope. So to configure a particular processor, configs could be added to
processors.processorId.*

I would like to discuss the "processors" scope further. I am planning on a proposal that doesn't require the user to specify a processorId. It is a hassle for the user to specify a unique ID for each processor within the job. I am saying this with the assumption that the relationship between StreamApplication and processors is 1:N --> 1 Streamapplication consists of 1 or more processors. Ideally, N is bounded by the number of partitions in the input streams and grouper criteria. 
I am guessing for multi-stage, you want to associate properties at every stage. If that is case, I would recommend a different approach of associating a group of processors to a stage identifier. This doesn't have to exposed to the user. It can be internally generated and managed. I would like to understand the requirements for this in the multi-stage scenario before I suggest any more solutions :D